src/main.d

```diff
@@ -41,6 +41,8 @@ int main(string[] args)
 	// Does the user want to disable upload validation - https://github.com/abraunegg/onedrive/issues/205
 	// SharePoint will associate some metadata from the library the file is uploaded to directly in the file - thus change file size & checksums
 	bool disableUploadValidation = false;
+	// Perform only a dry run - not applicable for --monitor mode
```

Comment:
I don't see where the `monitor` flag is checked against `dryRun`. It should be checked if it is present, shouldn't?

Reply:
As per readme:

`--dry-run` can only be used with `--synchronize`. It cannot be used with `--monitor` and will be ignored.